README.md

```diff
@@ -1,299 +1,64 @@
-<!-- 
-based on: https://gitlab.gfdl.noaa.gov/fre2/workflows/postprocessing/-/raw/d51df76e537222a3c78405b5749fe59306e6d2bd/README.md
--->
-This repository holds code for defining tasks, applications, tools, workflows, and other aspects of the FRE2 postprocessing
-ecosystem. 
+# fre-workflows
 
+This repository holds code for defining GFDL's next-generation FRE (FMS Runtime Environment) cylc workflows.
+Currently, this repository only contains the `fre-cli` postprocessing workflow.  This documentation will be updated
+when more workflows are added to this repository.
 
-<!-- ______________________________________________________________________________________________________________________ -->
-# Instructions to postprocess FMS history files on GFDL's PP/AN
-These instructions are targeted at workflow developers. If you are a user simply looking to run a specific workflow, see 
-[`fre-cli`](https://github.com/NOAA-GFDL/fre-cli). 
+For more information on cylc workflows please refer to
+[cylc's User Guide](https://cylc.github.io/cylc-doc/stable/html/user-guide/index.html)
```

Comment:
For more information on Cylc workflows, see [cylc's User Guide](https://cylc.github.io/cylc-doc/stable/html/user-guide/index.html)

this suggestion uses  less words and sounds more direct to me

Reply:
Actually I'm not sure if this line is needed at all.  Can the link be added to the above paragraph where Cylc is mentioned?